docs/index.rst

```diff
@@ -3,22 +3,49 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to cinnabar's documentation!
-=========================================================
+======================================
+Welcome to the cinnabar documentation!
+======================================
+
+``cinnabar`` is an open-source framework within the Open Free Energy (OpenFE) eco-system which enables consistent and
```

Comment:
Might be good somewhere here to mention that it used to be called `arsenic`, because they'll have seen references to arsenic in the literature.